### PR TITLE
Move all the apps into a Kubernetes cluster

### PR DESCRIPTION
**Pull request by Katharine**
Currently everything lives on an AppEngine instance somewhere. Move them all into a Kubernetes cluster and include the configuration here. Also, for good measure, include the slackin configuration as currently running at slack.k8s.io.

This code and configuration is currently live and serving production.

(The underlying motivation here is being able to make changes to the moderation bot that would make its cold start time less than stellar.)

/cc @BenTheElder

### CODE REVIEW

File: slack-event-log/main.go [hunk 1]
```diff
@@ -38,10 +38,15 @@ func parseFlags() options {
 	return o
 }
 
+func handleHealthz(w http.ResponseWriter, r *http.Request) {
+	_, _ = w.Write([]byte("ok"))
```
Comment (BenTheElder) [inline slack-event-log/main.go:42]: the best kind of healthz

**Review by BenTheElder — APPROVED** [review]
/lgtm

File: slack-event-log/main.go (hunk 1 above)
Comment (Katharine) [reply slack-event-log/main.go:42]: :D

**Pull request merged**